devel/api.md

```diff
@@ -655,6 +661,57 @@ Type: `string`
 
 The interface name which current VLAN is based on.
 
+# VXLAN
+Besides basic interface properties, each VXLAN interface state also contains
+a dictionary saved in key `VXLAN.CONFIG_SUBTREE`.
+
+Example:
+
+```python
+{
+    Interface.KEY: [
+        Interface.NAME: "eth1.101",
+        Interface.TYPE: VXLAN.TYPE,
+        Interface.STATE: InterfaceState.UP
+        VXLAN.CONFIG_SUBTREE: {
+            VXLAN.ID: 101,
+            VXLAN.BASE_IFACE: 'eth1'
+            VXLAN.REMOTE: '192.168.3.3'
+        }
+    ]
+}
+```
+
+## `VXLAN.ID`
+
+Type: `integer`
+
+The VXLAN ID.
+
+To change VXLAN ID, please use `InterfaceState.ABSENT` for existing one and add
+new one after that.
+
+
+## `VXLAN.BASE_IFACE`
+
+Type: `string`
+
+The interface name for VXLAN underlay.
+
+
+## `VXLAN.REMOTE`
+
+Type: `string`
+
+Specifies the unicast destination IP address to use in outgoing packets when the destination link layer address is not known in the VXLAN device forwarding database, or the multicast IP address to join.
```

Comment:
To make it similar to the other entries, the sentence should not start with `Specifies` but with `The unicast destination address...`